modules/server/src/main/ts/bedrock/server/Routes.ts

```diff
@@ -15,6 +16,20 @@ const createServer = (root: string) => {
   return server(root) as (request: IncomingMessage, response: ServerResponse, done: (err?: any) => void) => void;
 };
 
+const doResponse = (request: IncomingMessage, response: ServerResponse, status: number, contentType: string, data: any) => {
+  response.setHeader('ETag', RouteUtils.generateETag(data));
+  if (status === 304 || RouteUtils.isCachable(status) && RouteUtils.isFresh(request, response)) {
+    response.writeHead(304, { });
+    response.end();
+  } else {
+    response.writeHead(status, {
+      'Content-Type': contentType,
+      'Cache-Control': 'public, max-age=0'
```

Comment:
Why did we choose max-age over no-cache?

Reply:
Answering myself:
https://stackoverflow.com/a/1383359/7377237
Seems that max-age allows the browser to use a stale copy where as no-cache requires the browser to always fetch a new copy which might be a little too aggressive.

Reply:
To be completely honest I did this, as that's what `serve-static` was using, so I just wanted to keep it consistent.